logbook-json/src/main/java/org/zalando/logbook/json/StaticParameterReplacementOperator.java

```diff
@@ -0,0 +1,17 @@
+package org.zalando.logbook.json;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+
+import java.util.function.BinaryOperator;
+
+@AllArgsConstructor
+@EqualsAndHashCode
+final class StaticParameterReplacementOperator<T> implements BinaryOperator<String> {
```

Comment:
Same here. `T` is always `String`, no need to be generic.

Reply:
Not in this case. It could be number or boolean.